src/neuralnet/researcher.cpp

```diff
@@ -770,11 +831,16 @@ MiningProject MiningProject::Parse(const std::string& xml)
         ExtractXML(xml, "<team_name>", "</team_name>"),
         ExtractXML(xml, "<master_url>", "</master_url>"));
 
+    if (IsPoolCpid(project.m_cpid) && !GetBoolArg("-pool", false)) {
```

Comment:
Could we change the flag from `-pool` to something like `-pooloperator` or`-poolowner`? A user looking over the command line arguments help message (if/when added there) might think that the flag is for pool users instead of pool operators

Reply:
That's a good idea. I'll change it to `-pooloperator`.